index.js

```diff
@@ -549,6 +602,16 @@ function getRedirectUrl (url) {
   }
 }
 
+function generateFileHash (filePath) {
+  const hashSum = crypto.createHash('md5')
+
+  try {
+    const fileBuffer = readFileSync(filePath)
+    hashSum.update(fileBuffer)
+    return hashSum.digest('hex').slice(0, 16)
+  } catch {}
```

Comment:
this should either throw or the error should be handled.

Reply:
If not handled the has will be undefined, I guess we could make it an empty string so that if there’s an error, we access the normal location of the file

Reply:
done